src/lavinmq/mqtt/client.cr

```diff
@@ -135,11 +135,17 @@ module LavinMQ
 
       def details_tuple
         {
-          vhost:        @broker.vhost.name,
-          user:         @user.name,
-          protocol:     "MQTT",
-          client_id:    @client_id,
-          connected_at: @connected_at,
+          vhost:             @broker.vhost.name,
+          user:              @user.name,
+          protocol:          "MQTT 3.1.1",
+          client_id:         @client_id,
+          name:              @name,
+          connected_at:      @connected_at,
+          state:             state,
+          ssl:               @connection_info.ssl?,
+          tls_version:       @connection_info.ssl_version,
+          cipher:            @connection_info.ssl_cipher,
+          client_properties: AMQP::Table.new,
```

Comment:
Does this really need to be a Table? It's not possible to just put an empty named tuple here?
If it must be a Table it's probably better to have one Table instance as a class variable and use that as an "empty" Table.

Reply:
no, you're right! lets use empty NamedTuple